group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -701,6 +726,9 @@ private void removeGroup(
                     ClassicGroup classicGroup = (ClassicGroup) group;
                     metrics.onClassicGroupStateTransition(classicGroup.currentState(), null);
                     break;
+                default:
+                    log.warn("Removed group {} with an unknown group type {}. " + groupId, group.type());
```

Comment:
nit: There is an extra space after the `.` and the `+` must be replaced by a `,`.